collect.py

```diff
@@ -7,6 +7,18 @@
 from selenium import webdriver
 from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
 
+from sys import platform as _platform
+if _platform == "linux" or _platform == "linux2":  # linux
+    chromebin = "tools/chrome-linux/chrome"
```

Comment:
Rename `chromebin` and `nightlybin` to `chrome_bin` and `nightly_bin`.

Reply:
okay